docs/source/user/settings.rst

```diff
@@ -432,7 +432,7 @@ be used.
 .. code-block:: python
 
     {
-        'default': 'https://{site}/{organization}/password/reset/confirm/{uid}/{token}'
+        '__all__': 'https://{site}/{organization}/password/reset/confirm/{uid}/{token}'
```

Comment:
We also need to update this example https://github.com/openwisp/openwisp-radius/blame/f296f237983c51cb70b18f2329dba039cda9799f/docs/source/user/settings.rst#L464-L468